### PR TITLE
Update comparison code related to change in `pm_tb_data`

### PR DESCRIPTION
**Pull request by trey-stafford**
Small tweak to the comparison code to support changes being made in `pm_tb_data`.

To think about: should we keep this visualization code around? Remove it? Most of this was crafted for stakeholder meetings to show progress. 

Some code to visualize PM data, sea ice concentration data w/ and w/o flags, etc. would be nice to have. Maybe we can keep some of this code in the `pm_tb_data` or `seaice_ecdr` package? Or a separate repo?

### CODE REVIEW

**Review by sc0tts — APPROVED** [review]

**Comment by trey-stafford** [comment]
Just waiting on this to get merged before we merge this PR: https://github.com/nsidc/pm_tb_data/pull/9/

**Pull request merged**
